tests/compile_tests/midi/relative_to_include.bas

```diff
@@ -0,0 +1,11 @@
+$CONSOLE:ONLY
+OPTION _EXPLICIT
+
+'$Include:'include/soundfont.bi'
+
+DIM h AS LONG: h = _SNDOPEN("2SKIPIX.mid")
```

Comment:
If we include the MIDI file and run and get the output, we can also test if the SF3 stuff is working or not.

Reply:
Yeah good point, I should have just left it and added `.output` files :D